mat/symband.go

```diff
@@ -14,6 +14,7 @@ var (
 	_            Matrix           = symBandDense
 	_            Symmetric        = symBandDense
 	_            Banded           = symBandDense
+	_            SymBanded        = symBandDense
```

Comment:
Diagonal should also have the methods added to satisfy this

Reply:
I think that's another PR.